LiteCore/Support/config.h.in

```diff
@@ -0,0 +1,2663 @@
+/**
```

Comment:
This file has a really generic name and location, making it hard to tell what it's for (you have to read down to line 25.) Also, it's not a source or header file, so I don't think it belongs directly in Support. Could it be moved to some mbed-specific directory?